pywr/parameters/_hydropower.pyx

```diff
@@ -23,17 +23,17 @@ cpdef double inverse_hydropower_calculation(double power, double water_elevation
         An efficiency scaling factor for the power output of the turbine.
     flow_unit_conversion : double (default=1.0)
         A factor used to transform the units of flow to be compatible with the equation here. This
-        should convert flow to units of $m^3/day$
+        should convert flow to units of $m^3/second$
     energy_unit_conversion : double (default=1e-6)
-        A factor used to transform the units of power. Defaults to 1e-6 to assuming input of $MJ$/day. 
```

Comment:
@jetuk Do you have an opinion on the units we should be using?

Reply:
MegaWatt (MW) is the unit commonly used for power

Reply:
Yeah, MW probably makes a better default to MJ.